docs/source/developer/contributing.rst

```diff
@@ -270,6 +270,9 @@ There is potential for confusion between the Numba module ``typing`` and Python
 hints, as well as between Numba types---such as ``Dict`` or ``Literal``---and ``typing`` types of the same name.
 To mitigate the risk of confusion we use a naming convention by which objects of the built-in ``typing`` module are
 imported with an ``pt`` prefix. For example, ``typing.Dict`` is imported as ``from typing import Dict as ptDict``.
+Similarly, the Python built-in ``typing`` module is imported as ``import typing as pt``.
```

Comment:
I think we should move everything to the `pt.` naming, instead of of the `pt` prefix I used. what do you think?

Reply:
Agreed